docs/tutorials/tpu-cluster-setup.md

```diff
@@ -0,0 +1,135 @@
+# Setting up a TPU Cluster
+
+This guide will walk you through the steps to set up and manage a TPU cluster using Google Cloud Platform (GCP) and Ray.
```

Comment:
Say this is needed to use Marin to train on TPUs?

Reply:
I think we can train on a standalone TPU too. We don't necessarily need to spin up a Ray cluster. @dlwh, Am I correct about this? Or should I add this line?

I can add, something like:

"You would need this setup to perform large scale runs" ?

Reply:
only on a v4-8 or similar. multinode (v4-16 etc) Marin isn't going to work without the autoscaler as currently set up. We could make it work but it is work